mylasqlanon.py

```diff
@@ -127,21 +130,27 @@
     # These methods are based on using another column as an index 
     # and applying the changes in bulk rather than individually
     for col in t_config:
-        mod_name, index_name = (col.get('method').rsplit('.', 1) + [None] * 2)[:2]
-        col_name = col.get('name')
+        mod_name = col.get("module")
+        method_name = col.get("method")
+        index_name = col.get("index")
+        col_name = col.get("name")
+        source_name = col.get("source")
+        if mod_name and method_name:
+            dyn_func = getattr(locals().get(mod_name), method_name)
```

Comment:
This dynamic function idea is cool, but perhaps makes the code a little more difficult to read. Maybe we could just add a debug log message somewhere here that provides the function name?